_sidebar.md

```diff
@@ -16,12 +16,10 @@
 * [[Getting Started]] 🙌
 * [[NuGet Packages]] 📦
 * [[Preview Packages]] 🌙
-* Have [[Questions]] ❓
```

Comment:
I did like having questions in both places, as I still see this section for developers using the toolkit (and introductions to the project in the top section) vs. moving into contributions and the project technical details in the sections below.

Reply:
I deleted that just to avoid redundancy as it looks a little off where we have the same page in two different sections. I do agree it becomes more user friendly to keep it in the usage section vs contribution. 

Plan 1: We can keep it in the Usage section and delete it from Contribution.  
Plan 2: We can keep it in both. 

Totally up to you.

Reply:
Let's have it a single time in the usage section then! Thanks!